### ISSUE TITLE
Cannot move templates to new folder via VoiceOver

### ISSUE THREAD

**Issue by heyitsmebev** [GSA/notifications-admin#2057]
New folder button and the steps to create a new folder seems to work well. However, since I cannot check one of the current templates, I cannot move the template to the folder. Cancel and Add new folder works. 
- [ ] Cannot move templates to new folder
![Image](https://github.com/user-attachments/assets/79890441-1991-4f05-a744-385df5992294)

_Originally posted by @heyitsmebev in https://github.com/GSA/notifications-admin/issues/1940#issuecomment-2429853746_
